components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/util/OAuth2Util.java

```diff
@@ -1677,4 +1768,39 @@ private static String hexify(byte bytes[]) {
         return essentailClaimslist;
     }
 
+    public static String getSanitizedUserStoreDomain(String userStoreDomain) {
+        if (userStoreDomain != null) {
```

Comment:
What about using StringUtils.isNotBlank(userStoreDomain)  ?

Reply:
This was an existing method, even the PR showed as a new method. Anyway its good to improve the method as you mentioned. So will fix that.